examples/starrocks/deploy_a_starrocks_cluster_with_all_features.yaml

```diff
@@ -0,0 +1,174 @@
+# This manifest deploys a StarRocks cluster with all supported features.
+# If you want to understand the meaning of the fields in the following custom resource, please
+# refer to the explanations in other examples.
+
+apiVersion: starrocks.com/v1
+kind: StarRocksCluster
+metadata:
+  name: starrockscluster-sample
```

Comment:
make difference of each example, so we may get clue on what example the user is using.